lms/envs/production.py

```diff
@@ -951,6 +951,12 @@ def get_env_setting(setting):
 ######################## Setting for content libraries ########################
 MAX_BLOCKS_PER_CONTENT_LIBRARY = ENV_TOKENS.get('MAX_BLOCKS_PER_CONTENT_LIBRARY', MAX_BLOCKS_PER_CONTENT_LIBRARY)
 
+######################## Setting for django-countries ########################
+# django-countries provides an option to make the desired countries come up in
+# selection forms, if left empty countries will come up in ascending order as before
+# https://github.com/SmileyChris/django-countries#show-certain-countries-first
+COUNTRIES_FIRST = ENV_TOKENS.get('DJANGO_COUNTRIES_FIRST', [])
```

Comment:
I believe using `ENV_TOKENS` is not required anymore - it happens automatically because of this line: https://github.com/edx/edx-platform/blob/78a5d001262be4d584150570c1790e9c8818d48c/lms/envs/production.py#L93

See also this comment:

https://github.com/edx/edx-platform/blob/78a5d001262be4d584150570c1790e9c8818d48c/lms/envs/production.py#L71

So you only need to add the setting to `common.py` and should still be able to override it via `lms.yml`

Reply:
This is awesome, and something completely new to me! Thanks for the heads up @bradenmacdonald.

Reply:
Thanks for pointing it out @bradenmacdonald, will make this change 😄